commands/commands.go

```diff
@@ -85,7 +85,7 @@ func FullError(err error) {
 
 func errorWith(err error, fatalErrFn func(error, string, ...interface{}), errFn func(string, ...interface{})) {
 	if Debugging || errors.IsFatalError(err) {
-		fatalErrFn(err, "")
+		fatalErrFn(err, "%s", err)
```

Comment:
That's a fair point too. I don't feel that strongly either way, perhaps just make it clearer in the doc for `LoggedError`?